client/src/containers/ContactsList/index.js

```diff
@@ -1,53 +1,46 @@
-import React, {useState} from "react";
+import React, { useState, useEffect } from "react";
 import PropTypes from "prop-types";
-import  InvitationDialog  from "../InvitationDialog";
+import InvitationDialog from "../InvitationDialog";
 import { useStyles } from "./style";
 import { List, Typography, Button } from "@material-ui/core";
 import ContactItem from "../../components/ContactItem";
 import AddIcon from "@material-ui/icons/Add";
+import {
+  useContactsDispatch,
+  useContactsState,
+  fetchContacts,
+  deleteContact,
+} from "../../context/contacts/contactsContext";
+const jwt_decode = require("jwt-decode");
```

Comment:
```suggestion
const jwtDecode = require("jwt-decode");
```

to stay consistent with snakeCase

Reply:
Changed variables to camel case style. Invitation list fetching will be implemented in task no35 "IN: Contact List".